src/test/ui/consts/offset_from_ub.rs

```diff
@@ -1,5 +1,8 @@
 #![feature(const_raw_ptr_deref)]
 #![feature(const_ptr_offset_from)]
+#![feature(core_intrinsics)]
+
+use std::intrinsics::ptr_offset_from;
```

Comment:
I changed this test to call the intrinsic directly, so that we can properly match on the spans for the error messages.